task2-bash/backup.sh

```diff
@@ -1,19 +1,19 @@
 #!/bin/bash
```

Comment:
As for me, the previous version was fine. You've truncated some useful information